tests/components/soundtouch/test_config_flow.py

```diff
@@ -0,0 +1,113 @@
+"""Test config flow."""
+from unittest.mock import patch
+
+from requests import RequestException
+from requests_mock import ANY, Mocker
+
+from homeassistant.components.soundtouch.const import DOMAIN
+from homeassistant.components.zeroconf import ZeroconfServiceInfo
+from homeassistant.config_entries import SOURCE_USER, SOURCE_ZEROCONF
+from homeassistant.const import CONF_HOST, CONF_SOURCE
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+from .conftest import DEVICE_1_ID, DEVICE_1_IP, DEVICE_1_NAME
+
+
+async def test_user_flow_create_entry(
+    hass: HomeAssistant, device1_requests_mock_standby: Mocker
+) -> None:
+    """Test the full manual user flow from start to finish."""
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN,
+        context={CONF_SOURCE: SOURCE_USER},
+    )
+
+    assert result.get("type") == FlowResultType.FORM
+    assert result.get("step_id") == "user"
+    assert "flow_id" in result
+
+    with patch(
```

Comment:
The async_setup_entry were not patched, so your tests were trying to make requests to soundtouch devices.

Reply:
Fixed in e42bef8140eb83a0863562fc26e2aabc47a89c2a

Reply:
Awesome, thank you!